react/package.json

```diff
@@ -68,24 +69,23 @@
     "@babel/preset-react": "^7.10.1",
     "@massds/mayflower-assets": "^0.2.2",
     "@proscom/gulp-svgr": "^0.1.3",
-    "@storybook/addon-a11y": "^5.3.18",
-    "@storybook/addon-actions": "^5.3.18",
-    "@storybook/addon-info": "^5.3.18",
-    "@storybook/addon-knobs": "^5.3.18",
-    "@storybook/addon-links": "^5.3.18",
-    "@storybook/addon-storysource": "^5.3.18",
-    "@storybook/addon-viewport": "^5.3.18",
-    "@storybook/addons": "^5.3.18",
+    "@storybook/addon-a11y": "^6.0.21",
+    "@storybook/addon-actions": "^6.0.21",
+    "@storybook/addon-controls": "^6.0.21",
+    "@storybook/addon-docs": "^6.0.21",
+    "@storybook/addon-knobs": "^6.0.21",
+    "@storybook/addon-links": "^6.0.21",
+    "@storybook/addon-storysource": "^6.0.21",
+    "@storybook/addon-viewport": "^6.0.21",
+    "@storybook/addons": "^6.0.21",
+    "@storybook/cli": "^6.0.21",
+    "@storybook/codemod": "^6.0.21",
     "@storybook/preset-scss": "^1.0.2",
-    "@storybook/react": "^5.3.18",
-    "@storybook/source-loader": "^5.3.18",
+    "@storybook/react": "^6.0.21",
+    "@storybook/source-loader": "^6.0.21",
     "@svgr/cli": "^5.4.0",
     "@svgr/plugin-jsx": "^5.4.0",
     "@svgr/webpack": "^5.4.0",
-    "acorn": "^5.7.2",
```

Comment:
The `acorn` package was only used for backstop. Using the storybook extract command means we no longer need to use acorn and its packages.

Reply:
what does storybook extract command do?